k8s/templates/business-create-ui.pipeline.json

```diff
@@ -72,7 +72,7 @@
             "displayName": "Git Repo URL",
             "description": "The URL to your GIT repo.",
             "required": true,
-            "value": "https://github.com/bcgov/lear.git"
+            "value": "https://github.com/bcgov/bcrs-business-create-ui"
```

Comment:
Would this be a good time to rename the repo "business-create-ui" ?

Reply:
Yes if that's what's happening. I don't know who gets to make that decision and I don't think the name matters very much, but if it's going to change eventually then this will also have to change

Reply:
The team can make that decision. It just has to be communicated and done.
This is kind of out of scope for this ticket/PR but then again, it might be a good time to change it.

Reply:
I'll look into it, but it will be separate from this PR. Carry on :)